WTWSMS/decisions/VIET_roleplay_decisions.txt

```diff
@@ -12,7 +12,6 @@ decisions = {
 		potential = {
 			is_ruler = yes
 			is_landed = yes
-			ai = no
```

Comment:
Just because it doesn't exist for you personally doesn't mean it doesn't exist for others.